great_expectations/cli/batch_request.py

```diff
@@ -129,6 +134,14 @@ def select_data_connector_name(
     if len(available_data_asset_names_by_data_connector_dict) == 1:
         return list(available_data_asset_names_by_data_connector_dict.keys())[0]
 
+    elif len(available_data_asset_names_by_data_connector_dict) == 2:
```

Comment:
Since `len(available_data_asset_names_by_data_connector_dict)` is used a fair bit, what are your thoughts on saving it as a var (perhaps `num_names`)?